kernel/src/main/java/org/cristalise/kernel/process/StandardServer.java

```diff
@@ -110,13 +110,8 @@ public static void standardInitialisation(Properties props, ResourceLoader res)
 
             AbstractMain.shutdown(0);
         }
-        else if (Gateway.getProperties().containsKey(AbstractMain.MAIN_ARG_SKIPBOOTSTRAP)) {
-            //minimum initialisation only
-            Bootstrap.init();
-        }
         else {
-            //initialisation and complete checking bootstrap & module items
-            Bootstrap.run();
+            Gateway.runBoostrap();
```

Comment:
`Bootstrap.init()` and `Bootstrap.run()` could now be changed to package-private?

Reply:
ok